modules/tinymce/src/plugins/link/test/ts/browser/AttributesPostprocessTest.ts

```diff
@@ -0,0 +1,64 @@
+import { ApproxStructure, FocusTools, Waiter } from '@ephox/agar';
+import { describe, it, before, after } from '@ephox/bedrock-client';
+import { Fun } from '@ephox/katamari';
+import { SugarDocument } from '@ephox/sugar';
+import { TinyAssertions, TinyHooks, TinySelections } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+import Plugin from 'tinymce/plugins/link/Plugin';
+
+import { TestLinkUi } from '../module/TestLinkUi';
+
+describe('browser.tinymce.plugins.link.AttributesPostprocess', () => {
+  const newRel = Fun.constant('noopener noreferrer');
+  const attrsOverride = (attrs: any) => {
+    attrs.rel = newRel();
```

Comment:
This only tests `rel` we should make sure that we can overrride all the attributes that it has so maybe a separate test that say adds something to every obj property.

- [ ] Add tests for more properties like `href` etc and make sure that the `data-mce-href` gets properly updated